src/main/java/io/tackle/pathfinder/controllers/AssessmentsResource.java

```diff
@@ -77,4 +70,15 @@ public Response deleteAssessment(@NotNull @PathParam("assessmentId") Long assess
     return Response.ok().status(Response.Status.NO_CONTENT).build();
   }
 
+  @GET
+  @Path("risks")
+  @Produces("application/json")
+  public List<RiskLineDto> getIdentifiedRisks(@NotNull @Valid @QueryParam("applicationId") List<Long> applicationList) {
```

Comment:
As mentioned in my original comment I think having the list of applications using QueryParams has limitations on the length of the URL.

Now it seems more clear that the user won't select one by one a set of applications but will use a filter toolbar located inside the "Reports" page; having said that, I think there is a potential risk since the set of applications to be processed can easily grow to make the URL too long. I think we can change this endpoint to become a "POST" and receive the list of applications inside the body of the request, what do you think?


https://user-images.githubusercontent.com/2582866/120456888-4d0adb80-c396-11eb-9238-e24556e8dd4f.mp4

Reply:
:+1: